### DISCUSSION TITLE
Ribbon Button Margin between image and header text

### DISCUSSION THREAD

**Discussion by mwave** [fluentribbon/Fluent.Ribbon#1077] [Q&A] [answered]
Hello, search a solution to add space between Icon and label text in RibbonButton like a margin. Thanks for help.

**Comment by batzen** [comment, marked as answer]
Hi,
you can either overwrite the resource `Fluent.Ribbon.DataTemplates.Button.Header` if you want to change the header template of all your buttons or assign `HeaderTemplate` for a single button.
You can find the default template at https://github.com/fluentribbon/Fluent.Ribbon/blob/c71e82075bc6e46e6c56dc6147c45c626735ac90/Fluent.Ribbon/Themes/Controls/Button.xaml#L215
